atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -109,7 +109,9 @@ private AtlasDbConstants() {
     public static final long DEFAULT_SWEEP_PAUSE_MILLIS = 5 * 1000;
     public static final long DEFAULT_SWEEP_PERSISTENT_LOCK_WAIT_MILLIS = 30_000L;
     public static final int DEFAULT_SWEEP_DELETE_BATCH_HINT = 1_000;
-    public static final int DEFAULT_SWEEP_CANDIDATE_BATCH_HINT = 1024;
+    public static final int DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_CASSANDRA = 1;
+    public static final int DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_NON_CASSANDRA = 1024;
+    public static final int DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_MINIMUM = 1;
```

Comment:
Dont think we need a Minimum here, its kinda confusing. The reason the CLI has 1 is also because we want to ensure it runs for Cassandra.
 We can have a  `Math.min(DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_CASSANDRA, DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_NON_CASSANDRA)` for the default CLI batch size if you want to ensure that it does not cause unnecessary OOMs in any KVSs.

Reply:
yeah originally i had it that way, but the CLI references it in an annotation where the value must be a constant

Reply:
Lets have: 
public static final int DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_MINIMUM = Math.min(DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_CASSANDRA, DEFAULT_SWEEP_CANDIDATE_BATCH_HINT_NON_CASSANDRA) in `SweepCommand` or `AtlasDbConstants`, whichever  works?

Reply:
that doesn't work either, it actually needs to be a constant

Reply:
Cool, lets leave it as is as we will get rid of these soon after the Cassandra sweep perf improvements.